src/Morpho.sol

```diff
@@ -158,6 +158,9 @@ contract Morpho is IMorphoStaticTyping {
         idToMarketParams[id] = marketParams;
 
         emit EventsLib.CreateMarket(id, marketParams);
+
+        // Call to initialize the IRM in case it is stateful.
+        IIrm(marketParams.irm).borrowRate(marketParams, market[id]);
```

Comment:
maybe it's better now to call _accrueInterest?

Reply:
It will be skipped because `elapsed == 0`

Reply:
No, elapsed is very big instead (we would not have touched lastUpdated at all for the moment). In any case I agree that it's weird, let me think about a nice fix.

Reply:
pushed in #642